src/object/srv_obj.c

```diff
@@ -1492,7 +1492,7 @@ static void
 obj_ioc_fini(struct obj_io_context *ioc)
 {
 	if (obj_ioc_is_rebuild_container(ioc))
-		daos_csummer_destroy(&ioc->ioc_coh->sch_csummer);
+		daos_csummer_destroy(&ioc->ioc_coc->sc_csummer);
```

Comment:
Since it's per ds_cont_child now, it'll be destroyed when freeing ds_cont_child.  (on container destroy or server shutdown)

Reply:
removed